crates/cli/src/main.rs

```diff
@@ -1,10 +1,60 @@
 //! This is the CLI driver for the compilation of LLVM IR to Cairo. For more
-//! detail, please see the documentation for the [`hieratika_compiler`] crate.
+//! detail, please see the documentation for the [`hieratika_cairoc`] crate.
```

Comment:
This comment is not correct, and should also link to both the `hieratika_cairoc` and `hieratica_compiler` crates.

Reply:
I removed `hieratika_compiler` because the command has not been implemented yet.

Reply:
Ah okay. All good then!